ansible_base/authentication/authenticator_plugins/ldap.py

```diff
@@ -374,7 +374,7 @@ def authenticate(self, request, username=None, password=None, **kwargs) -> (obje
         # Ensure USER_SEARCH and GROUP_SEARCH are converted into a search object
         for field, search_must_have_user in [('GROUP_SEARCH', False), ('USER_SEARCH', True)]:
             data = getattr(self.settings, field, None)
-            if data is None:
+            if not data:
                 setattr(self.settings, field, None)
             elif not isinstance(data, config.LDAPSearch):
                 try:
```

Comment:
Hey Bryan, I added this comment to make the condition clearer. Handling [] or None is a common pitfall, and this should help avoid confusion. Just a small change to make the intent more obvious! 😊 Other than that, LGTM!

```python
# Ignore None or empty (e.g., [])
if not data:
    setattr(self.settings, field, None)

Reply:
Very nice, thanks!